{"uuid": "621c24f7-443c-453f-9d62-6ac7a8c92bf4", "children": ["f70056e1-6d71-454c-989e-5926eb810366", "a961cc8a-8c62-4288-9ee2-26aad1e541c4", "4a1e4a71-d3da-4abd-a9e2-7b56c94a6950", "147bf57d-0709-4e49-80cd-657ecb169347", "a1f08e14-6c12-4d0e-8bbc-51c4a1e7a93c", "e963cd7f-93d6-4b90-971c-3e83e2d268b7", "d749f3d5-4239-40c8-97f4-40632596b011", "f5e09633-df79-4ab8-be6a-877cad5e2c01"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1595337706286, "stop": 1595337706286}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\code_all\\\\AutoTest_all\\\\ATScripts-master\\\\report\\\\report.html'\n", "trace": "  File \"C:\\ProgramData\\Anaconda3\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\ProgramData\\Anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 796, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\ProgramData\\Anaconda3\\lib\\site-packages\\_pytest\\python.py\", line 726, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"C:\\ProgramData\\Anaconda3\\lib\\site-packages\\_pytest\\python.py\", line 660, in _call_with_optional_argument\n    func(arg)\n  File \"C:\\code_all\\AutoTest_all\\ATScripts-master\\scripts\\test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"C:\\code_all\\AutoTest_all\\ATScripts-master\\uti\\EmailHandler.py\", line 45, in send_email\n    send_content = self.read_report()  # 获取测试报告\n  File \"C:\\code_all\\AutoTest_all\\ATScripts-master\\uti\\EmailHandler.py\", line 20, in read_report\n    f = open(conf.TEST_CASE_REPORT_PATH, 'rb')\n"}, "start": 1595337745025, "stop": 1595337748509}], "start": 1595337706286, "stop": 1595337748510}